### ISSUE TITLE
Failed login session error

### ISSUE THREAD

**Issue by setu1421** [chroa/twitter-connect-plugin#25]
I have followed all the steps mentioned in the installation guide but got failed login session error while trying to connect from Ionic app. I have followed https://ionicframework.com/docs/native/twitter-connect/ tutorial to do that.

Why this error is coming up?

**Comment by hany0526** [comment]
have the same error

**Comment by shawns582** [comment]
@setu1421 @hany0526 Im having the same issue now (it was previously working). Have either of you found a solution?

**Comment by setu1421** [comment]
@shawns582 , No, i haven't found any solution :(

**Comment by tsjmattick** [comment]
I am getting the same thing after I moved this code into another project.

**Comment by switchstance-devs** [comment]
Adding your Twitter consumer and secret keys in below format along other preferences in config.xml file solves the issue:

`<preference name="TwitterConsumerKey" value="twitter key" />
<preference name="TwitterConsumerSecret" value="twitter secret" />`

**Comment by hany0526** [comment]
@switchstance-devs Thanks

**Comment by shawns582** [comment]
Yes I actually already had that in my config.xml from before (when the
plugin was working)

**Comment by AshokNaik009** [comment]
Facing the same issue even after adding
```
<preference name="TwitterConsumerKey" value="twitter key" />
<preference name="TwitterConsumerSecret" value="twitter secret" />
```
in config.xml has anyone solved it
![image](https://user-images.githubusercontent.com/29699443/121796495-0d19e300-cc37-11eb-8529-6547084e880a.png)
